debugger/src/dconfig.c

```diff
@@ -308,11 +318,20 @@ static gpointer saving_thread_func(gpointer data)
 			debug_config_changed = FALSE;
 		}
 
+#if GTK_CHECK_VERSION(3, 0, 0)
+		interval = g_get_monotonic_time() + SAVING_INTERVAL;
+#else
 		g_get_current_time(&interval);
 		g_time_val_add(&interval, SAVING_INTERVAL);
+#endif
 	}
+#if GTK_CHECK_VERSION(3, 0, 0)
+	while (!g_cond_wait_until(&cond, &change_config_mutex, interval));
+	g_mutex_unlock(&change_config_mutex);
```

Comment:
This requires glib 2.32 at least, not gtk3.

Reply:
Sorry, I mean ```g_cond_wait_until()``` replacing ```g_cond_timed_wait```.

Reply:
Fixed: the old code has been replaced with the new one.